tests/test_models/test_nn/deepar_native/test_deepar_native.py

```diff
@@ -0,0 +1,108 @@
+from unittest.mock import MagicMock
+
+import numpy as np
+import pytest
+from pytorch_lightning import seed_everything
+
+from etna.metrics import MAE
+from etna.models.nn import DeepARNativeModel
+from etna.models.nn.deepar_native.deepar import DeepARNativeNet
+from etna.models.nn.deepar_native.loss import GaussianLoss
+from etna.models.nn.deepar_native.loss import NegativeBinomialLoss
+from etna.pipeline import Pipeline
+from etna.transforms import StandardScalerTransform
+from tests.test_models.utils import assert_model_equals_loaded_original
+from tests.test_models.utils import assert_sampling_is_valid
+
+
+@pytest.mark.parametrize(
+    "horizon,loss,transform,epochs,lr,eps",
+    [
+        (8, GaussianLoss(), [StandardScalerTransform(in_column="target")], 100, 1e-3, 0.05),
+        (13, GaussianLoss(), [StandardScalerTransform(in_column="target")], 100, 1e-3, 0.05),
+        (15, GaussianLoss(), [StandardScalerTransform(in_column="target")], 100, 1e-3, 0.05),
+        (8, NegativeBinomialLoss(), [], 300, 1e-2, 0.05),
```

Comment:
How did you come to increasing the `lr` for `NegativeBinomialLoss`?

Reply:
tried to fix high metrics
one of the way to overfit more without increasing test time - increase lr

Reply:
Not sure that it is the way to overfit -- more looks tile the way to not converge